pydm/widgets/image.py

```diff
@@ -25,8 +35,14 @@ class PyDMImageView(ImageView, PyDMWidget, PyDMColorMap):
         The channel to be used by the widget to receive the image width
         information
     """
-    
+
+    Q_ENUMS(_ReadingOrderMap)
     Q_ENUMS(PyDMColorMap)
+
+    readingorderdict = {}
+    for rd, i in READINGORDER.items():
```

Comment:
Please use the same structure as the Label for the ENUMS...
Ref:
https://github.com/slaclab/pydm/blob/master/pydm/widgets/label.py#L8
https://github.com/slaclab/pydm/blob/master/pydm/widgets/display_format.py#L8

Reply:
Only if this will be used in more than one place...
The `DisplayFormat` is used at least on the label and lineEdit... 
If this is the only place feel free to drop the code at the `image.py`